keyvault/data-plane/azure-keyvault/src/main/java/com/microsoft/azure/keyvault/requests/CreateCertificateRequest.java

```diff
@@ -74,6 +74,10 @@ public Builder withPolicy(CertificatePolicy certificatePolicy) {
          * @return the Builder object itself.
          */
         public Builder withAttributes(Attributes attributes) {
+            if (!(attributes instanceof CertificateAttributes)) {
+                throw new IllegalArgumentException("Parameter attributes should be instance of CertificateAttributes");

```

Comment:
What do you think about saying 'Parameter attributes should be instance of CertificateAttributes, **or a subclass**'? Also, what do you think about putting apostrophe's around the variable name?

Reply:
Sounds good. I'll fix it.